cmd/version.go

```diff
@@ -21,7 +21,7 @@ func init() {
 	versionCmd.PersistentFlags().BoolVarP(
```

Comment:
```suggestion
	versionCmd.PersistentFlags().BoolVar(
```

Can we drop the shorthand flag instead?